apisix/ssl.lua

```diff
@@ -86,32 +101,32 @@ function _M.aes_encrypt_pkey(origin)
 end
 
 
-local function decrypt_priv_pkey(iv, key)
-    local decoded_key = ngx_decode_base64(key)
-    if not decoded_key then
-        core.log.error("base64 decode ssl key failed. key[", key, "] ")
-        return nil
+local function aes_decrypt_pkey(origin)
+    if core.string.has_prefix(origin, "---") then
+        return origin
     end
 
-    local decrypted = iv:decrypt(decoded_key)
-    if not decrypted then
-        core.log.error("decrypt ssl key failed. key[", key, "] ")
+    local aes_128_cbc_with_iv_tbl = get_aes_128_cbc_with_iv()
+    if core.table.isempty(aes_128_cbc_with_iv_tbl) then
```

Comment:
Use `#` would be better as the table must be array

Reply:
Done